docs/GettingStartedDocs/Contributors/WindowsManualInstallPrereqs.md

```diff
@@ -6,8 +6,7 @@
  Note: To check if your system has support for SGX1 with or without FLC, please look [here](../SGXSupportLevel.md).
  
 - A version of Windows OS with native support for SGX features:
-   - For server: Windows Server 2016
-   - For client: Windows 10 64-bit version 1709 or newer
+   - Windows Server 2016
    - To check your Windows version, run `winver` on the command line.
```

Comment:
Technically you don't need this instruction anymore since the `winver` check was largely for confirming that the Windows 10 client was on the right update version.

Reply:
Removed.